lib/apptimize_flutter.dart

```diff
@@ -558,7 +674,11 @@ class ApptimizeValueVariable<T> extends ApptimizeVariable<T> {
   /// Returns the default value if there is an issue with the incoming variant
   /// data.
```

Comment:
Should also mention the fact that it can now throw? or should a null value return the default?

Reply:
Turns out null values are allowed, so this is just wrong. I've changed it.